opm/autodiff/FlowMainEbos.hpp

```diff
@@ -559,12 +624,24 @@ namespace Opm
             // create output writer after grid is distributed, otherwise the parallel output
             // won't work correctly since we need to create a mapping from the distributed to
             // the global view
-            output_writer_.reset(new OutputWriter(grid(),
+            std::vector<double> global_perms;
+            const double* writer_perms = fluidprops_->permeability();
+
+            if ( must_distribute_ )
+            {
+                // Create global permeabilities.
+                global_perms.resize(this->globalGrid().size(0));
+                gatherCellDataToGlobalIterator(this->grid(),
+                                                writer_perms,
+                                                global_perms.data());
+                writer_perms = global_perms.data();
+            }
+            output_writer_.reset(new OutputWriter(globalGrid(),
                                                   param_,
                                                   eclState(),
                                                   std::move(eclipse_writer_),
                                                   Opm::phaseUsageFromDeck(deck()),
-                                                  fluidprops_->permeability()));
+                                                  writer_perms));
```

Comment:
a bit more principle question: wouldn't it be easier to just reuse the transmissibilities which need to be computed for load balancing than gathering them?